clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/client/Applications.java

```diff
@@ -580,6 +580,11 @@ public void updateProcess(
     if (healthCheckEndpoint != null && !healthCheckEndpoint.isEmpty() && healthCheck != null) {
       healthCheck.setData(new Process.HealthCheckData().setEndpoint(healthCheckEndpoint));
     }
+    if (command != null && command.isEmpty()) {
```

Comment:
it might be worth checking for `""`, in this case, it should just default to the original buildpack command

Reply:
@kevinawoo I believe "" is being checked by command.isEmpty()

Reply:
I feel like an empty string in the manifest is potentially problematic and wanted to throw something if thats found. For example, if there are variables being substituted but in that process of substitution it resolved to an empty string it would be good for the user to know.

Reply:
👍 makes sense, I'll leave this convo open incase someone comes looking to why `""` is throwing that error.

They might have a use case to further improve the error handling/message.